org.mpo.Dayon.yml

```diff
@@ -0,0 +1,53 @@
+app-id: ch.galante.Dayon
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: dayon_assistant.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=home:rw
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: dayon
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 dayon.jar -t /app/bin
+      - install -Dm755 dayon_*.sh -t /app/bin
+      - install -Dm755 dayon.flatpak.sh -t /app/bin/dayon.sh
+      - install -Dm755 dayon.browser -t /app/bin
+      - install -Dm644 dayon.png /app/share/icons/hicolor/128x128/apps/ch.galante.Dayon.png
+      - install -Dm644 dayon.assisted.desktop /app/share/applications/ch.galante.Dayon.desktop
+      - desktop-file-edit --set-key=Exec --set-value='/app/bin/dayon_assisted.sh' /app/share/applications/ch.galante.Dayon.desktop
+      - desktop-file-edit --set-icon=ch.galante.Dayon /app/share/applications/ch.galante.Dayon.desktop
+      - install -Dm644 dayon.assistant.desktop /app/share/applications/ch.galante.Dayon.assistant.desktop
+      - desktop-file-edit --set-key=Exec --set-value='/app/bin/dayon_assistant.sh' /app/share/applications/ch.galante.Dayon.assistant.desktop
+      - desktop-file-edit --set-icon=ch.galante.Dayon /app/share/applications/ch.galante.Dayon.assistant.desktop
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/RetGal/Dayon/releases/download/v11.0.5/dayon.11.0.5.tgz
+        sha256: deeb09fa9d84017af9f3289c7e11212388799e80d58703dc60f84cd6accb3838
+      - type: file
+        dest-filename: dayon.flatpak.sh
+        url: https://raw.githubusercontent.com/RetGal/Dayon/master/flatpak/dayon.sh
```

Comment:
My comment about not checkout from a branch (`msaster`) still stick. a specific revision / tag is required.

Also below